src/fontra/views/editor/editor.js

```diff
@@ -1411,9 +1411,41 @@ export class EditorController {
       { title: "Cancel", isCancelButton: true },
       { title: "Add", isDefaultButton: true, result: "ok", disabled: true },
     ]);
-    dialog.setContent(glyphsSearch);
+    dialog.setContent(
+      html.div(
+        {
+          style: `
+          grid-row: 2 / -1;
+          display: flex;
+          flex-direction: column;
```

Comment:
Please add some more space between the list and the checkbox. Compare with the space between the search field and the list:
<img width="396" alt="image" src="https://github.com/googlefonts/fontra/assets/4246121/21ca8554-d5b3-48dc-8199-2f52ce7d74b8">